client/shipping-label/views/steps/address/fields.js

```diff
@@ -3,8 +3,24 @@ import { translate as __ } from 'lib/mixins/i18n';
 import TextField from 'components/text-field';
 import CountryDropdown from 'components/country-dropdown';
 import StateDropdown from 'components/state-dropdown';
+import isObject from 'lodash/isObject';
+import isEqual from 'lodash/isEqual';
+import AddressSuggestion from './suggestion';
 
-const AddressFields = ( { values, allowChangeCountry, group, labelActions, storeOptions, errors } ) => {
+const AddressFields = ( { values, isValidated, normalized, pickNormalized, allowChangeCountry, group, labelActions, storeOptions, errors } ) => {
+	if ( isValidated && normalized && ! isEqual( normalized, values ) ) {
+		return (
+			<AddressSuggestion
+				values={ values }
+				normalized={ normalized }
+				pickNormalized={ pickNormalized }
+				pickNormalizedAddress={ ( pick ) => labelActions.pickNormalizedAddress( group, pick ) }
+				editOriginalAddress={ () => labelActions.editOriginalAddress( group ) }
```

Comment:
Another spot where `editAddress` would potentially be less likely to be confused with origin